falcon/asgi/ws.py

```diff
@@ -581,6 +600,13 @@ def __init__(self) -> None:
             1011: 'Internal Server Error',
             3011: 'Internal Server Error',
         }
+        for status_constant in dir(status_codes):
```

Comment:
I think it may make sense to move this to a method of the class

Reply:
Hm, just to make the code cleaner? Like `_create_default_close_reasons()`?

Reply:
that works yes.